tools/msstitch/ready/msslookup_isoquant.xml

```diff
@@ -0,0 +1,37 @@
+<tool id="msslookup_isoquant" name="Create lookup table with isobaric quant data">
+    <requirements>
+        <requirement type="package" version="2.2">msstitch</requirement>
+    </requirements>  
+  
+    <description>using spectra mzMLs and openMS isobaric quant</description>
+    
+    <command>
+            cp '$dbfile' spec_lookup.sql
+	    &&
```

Comment:
What the hell? I must have messed up in a git checkout from another package in the making...